domain-management/src/main/java/org/jboss/as/domain/management/parsing/AuditLogXml.java

```diff
@@ -46,17 +46,16 @@ public static AuditLogXml newInstance(Namespace namespace, boolean host) {
         }
     }
 
-    AuditLogXml() {
-    }
-
     // TODO - The problem is that it is version dependent but it would be nicer to not have processing instructions and find a
     // better way to decide if a native interface is required.
 
-    public abstract void parseAuditLog(final XMLExtendedStreamReader reader, final ModelNode address, final Namespace expectedNs, final List<ModelNode> list) throws XMLStreamException;
+    void parseAuditLog(final XMLExtendedStreamReader reader, final ModelNode address, final Namespace expectedNs, final List<ModelNode> list) throws XMLStreamException;
```

Comment:
I don't recall for sure, but this might fail the next checkstyle upgrade because the parameters use the `final` qualifier on an interface method.